### ISSUE TITLE
Collections landing/search page should display facets if configured to do so

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#532]
### Descriptive summary

After the collection landing/search page accepts facets, facets should display on the page, so people can... facet.

### Expected behavior

Facets display on collection landing/search page. Facets correctly facet search, per #531. Display and functionality should be parallel to the works search page.

### Related work

Depends on #531 
Part of #528 

### Accessibility Concerns

Facets added to display will add new HTML and possibly JS functionality. It should be the same as the works search pages, and ideally generated the same way so accessibility fixes applied to one affect the other.

**Comment by CGillen** [comment]
QA:
- [ ] Collections landing page offers facet for keyword metadata

**Comment by vantuyls** [comment]
QA pass - result are over there
